.gitignore

```diff
@@ -10,6 +10,5 @@ store
 store.js
 main.js
 extension.js
-*.png
```

Comment:
fyi, this was removed.
It didn't seem to affect any other files outside of docs/media